x-pack/legacy/plugins/apm/public/components/app/Settings/ApmIndices/index.tsx

```diff
@@ -94,11 +91,11 @@ export function ApmIndices() {
   const [apmIndices, setApmIndices] = useState<Record<string, string>>({});
   const [isSaving, setIsSaving] = useState(false);
 
-  const callApmApiFromHook = useCallApmApi();
-
   const { data = INITIAL_STATE, status, refetch } = useFetcher(
-    callApmApi =>
-      callApmApi({ pathname: `/api/apm/settings/apm-index-settings` }),
+    callApmApiFromFetcher =>
+      callApmApiFromFetcher({
```

Comment:
WDYT ?
```suggestion
    _callApmApi =>
      _callApmApi({
```

or perhaps

```suggestion
    api => api({...})
```